.changeset/rich-zoos-occur.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/backend-plugin-api': minor
+'@backstage/backend-defaults': minor
+---
+
+Removing shared environments concept from the new experimental backend system.
```

Comment:
This should be prefixed with `**BREAKING**:` and then let's highlight it in the release notes eventually in the mainline release.